ai/pocs/nn_layers/model_launcher.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+import json
+import sys
+
+def match_to_data(match):
+    first_x = []
+    second_x = []
+
+    for unit in match['first_player']['units']:
+        if unit is not None:
+            first_x.append(list(unit['stat'].values()))
+    for unit in match['second_player']['units']:
+        if unit is not None:
+            second_x.append(list(unit['stat'].values()))
+
+    sample = (
+        first_x,
+        second_x,
+        match['first_player']['score'],
+        match['second_player']['score']
+    )
+    return sample
+
+
+
+def format_value(value):
+    scores = []
+    final_len = 22
+    units = [np.array(value[0]), np.array(value[1])]
+    scores = np.array([value[2], value[3]])
+    units[0] = np.pad(units[0], ((0, final_len - len(units[0])), (0, 0)), 'constant')
+    units[1] = np.pad(units[1], ((0, final_len - len(units[1])), (0, 0)), 'constant')
+    return (np.array([units]), np.array(scores))
+
+
+
```

Comment:
Is it possible to put everything under the "if __name__" in an actual main definition ?
That is also related to my task, since we are not interpreting the python from rust but rather "compiling" it, I can only call actual definition from the files I load from python in rust